.github/workflows/label_merge_conflicts.yml

```
@@ -0,0 +1,23 @@
+name: Manage labels
```

```suggestion
name: Manage Labels
```
to match all the other workflows.